DMTN-272.tex

```diff
@@ -49,7 +50,184 @@
+
+More details about the exact data-set used for each experiment is included in the experiment's subsection.
+
+\begin{figure}[h!]
+  \centering
+  \includegraphics[width=0.7\textwidth]{exposure_hist.png}
+  \caption{Distribution of exposure numbers of the existing raw images of each of the tracts. Left: tract \#3080, right: tract \#4024.}
+  \label{fig:exposure_hist}
+\end{figure}
+
+\begin{figure}[h!]
+  \centering
+  %\includegraphics[width=0.7\textwidth]{tract_detectors.png}
+  \includegraphics[width=0.7\textwidth]{detector_hist.png}
+  \caption{Distribution of the detector indices corresponding to the raw images of each of the tracts. Left: tract \#3080, right: tract \#4024.}
+  \label{fig:detector_hist}
+\end{figure}
+
+\clearpage
+Template images were created per-tract, in the sense that the data input to the APTemplate pipeline was constrained to a single tract at each run. Figure~\ref{fig:tract_templates} illustrates an exemplar instance, where the effect of limiting the raws to a single tract apperas as croppings in the output templates. This is not of a concern in this very application though, since the downstream tasks receive proper masks and will generate cutouts only of the regions from which enough informative data can be extracted.
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=0.7\textwidth]{tract_templates.png}
+  \caption{Templates generated based on raw images from a single tract. The white chessboard in the background is the spatial spread of the detectors (a single visit/exposure), whereas the area ``full of stars'' represents a single tract in the sky.}
+  \label{fig:tract_templates}
+\end{figure}
+
+%----------------------------------------------------
+\clearpage
+\section{Evaluation Results}
+
+\subsection{The ``PCW 2022'' network and data}
+The data-set used for training this model, DL Dataset v0.1.1, consists of \texttt{78222} triplets, out of which \texttt{2591} triplets ($\sim 3\%$) are \texttt{positive}s.
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=0.5\textwidth]{precrec___checkpoint_epoch0000087_0200000____rbdata_data_npy_data_gt1M___posw_30.png}
+  \caption{No sigma-thresholding applied on the input sources (diffim output)}
+  \label{fig:tract_templates}
+\end{figure}
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=0.5\textwidth]{precrec___checkpoint_epoch0000087_0200000____rbdata_data_npy_data_gt1M-5sigma___posw_30.png}
+  \caption{Only sources with $snr > 5\sigma$}
```

Comment:
SNR here is the simulated SNR?  Or did you adjust the detection threshold of diffim?

Reply:
It is based on whatever was being reported under the SNR column at the time.